skyportal/tests/frontend/test_followup_requests.py

```diff
@@ -84,21 +84,17 @@ def add_followup_request_using_frontend_and_verify(
     try:
```

Comment:
This looks good, but can this just be a for loop with a break?

```
for i in range(2):
    ...
    if exception continue else break
```

Reply:
fixed